assets/css/free-trial-plan-picker-banner.css

```diff
@@ -2,13 +2,11 @@
 	background-color: #c9356e;
 	color: #ffffff;
 	font-size: 0.8125em;
-	position: fixed;
-	z-index: 9999;
+	position: sticky;
+	z-index: 99999;
 	width: 100%;
```

Comment:
```suggestion
```

On smaller screens (tested on iPhone XR) it seems there's a white spacing on the right caused by the banner:

<img width="475" alt="image" src="https://user-images.githubusercontent.com/3747241/230267873-bc8641f4-0b90-467f-a7a3-1a1a7891a2bb.png">

I find that removing width actually fixes it without any side effects. Let me know if this works!

<img width="458" alt="image" src="https://user-images.githubusercontent.com/3747241/230269836-d53098f9-8659-466a-a5d9-e6cf8afd86ec.png">